src/main/java/net/coderbot/patchwork/access/ModAccessTransformation.java

```diff
@@ -2,18 +2,18 @@
 
 import org.objectweb.asm.Opcodes;
 
-public class AccessTransformation {
-	public static final AccessTransformation NONE = new AccessTransformation(0, 0);
-	public static final AccessTransformation DEFINALIZE =
-			new AccessTransformation(Opcodes.ACC_FINAL, 0);
-	public static final AccessTransformation MAKE_PUBLIC =
-			new AccessTransformation(Opcodes.ACC_PRIVATE | Opcodes.ACC_PROTECTED,
+public class ModAccessTransformation {
```

Comment:
I don't think this needs to be renamed, it is intended to be generic to any access transformation.